content/blog/posts/2022-07-01-accessibility-basics.md

```diff
@@ -0,0 +1,153 @@
+  - '2022'
+  - web accessibility
+  - nuxt.js
+  - a11y
+date: '2022-07-01'
+slug: accessibility-basics
+title: 'Building (or Rebuilding) a Website with Accessibility in Mind'
+---
+
+Websites have come a long way since HTML and CSS were the only tools available to build them. So has design sense-- companies want to express their brand's identity using the latest technologies to create dynamic content that catches and maintains their users' attention.
+
+To ensure everyone is able to use your website, it is important to make sure the underlying structure of each page is set up in a way that works with the different devices and tools people may be using to interact with them.
+
+| ![AOL landing page from 1999, comprised of mostly low-resolution graphic ads and blue underlined links to different parts of the web](/img/blog/AOL-1999.png) |
+| :-----------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                    _Remember this beauty?_                                                                    |
+
+<br><br/>
+
+## Navigating a Modern Website with Accessibility Tools
+
+| ![shipshape.io landing page, newly updated this past year](/img/blog/shipshape-home.png) |
+| :--------------------------------------------------------------------------------------: |
+|                  _How do you process the content on this landing page?_                  |
+
+<br/>
+
+A visual flow that may instantly make sense for those of us with normal vision and the ability to navigate the GUI with a mouse or trackpad can be vastly different navigational experience for someone without those abilities.
+
+Some folks may rely on tools like a screenreader or keyboard to be able to access the digital world. These tools rely on software engineers to create apps with proper DOM structure so their users can to get to the information they need quickly and efficiently.
+
+## Generate the DOM Structure First
+
+<br/>
+
+![shipshape.io landing page marked up with arrows starting at the header and zig-zagging across the page to each text area](/img/blog/shipshape-home-arrows.png)
+
+If you are not visually impaired, chances are likely that your eyes jumped to scan each header on the page, starting at the top left. You might have next read a section of body copy, if the header caught your interest. Though your focus may have been on the text, your eyes concurrently processed the two images of our mascot at work "building" apps, which are opposite each segment of copy. You could likely skim through the buttons made up of tech logos on the bottom right-hand corner of the page.
+
+Someone without those same abilities could have a fairly similar experience if the page is built with their experience in mind. Based on our abilities as able-bodied developers, our instinct may be to add elements as we see them on a design spec.
```

Comment:
```suggestion
Someone without those same abilities could have a fairly similar experience if the page is built with their experience in mind. Based on our abilities as developers without a visual disability, our instinct may be to add elements as we see them on a design spec.
```

Reply:
Because the article content is specific to people with a visual disability I checked [this guide](https://www.acedisability.org.au/information-for-providers/language-disability.php) for preferred language. Might be useful where you see fit!